rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/OneScanCovidRDTBarcodeFactory.java

```diff
@@ -25,6 +26,7 @@ protected void launchBarcodeScanner(Activity activity, MaterialEditText editText
         InputMethodManager inputManager = (InputMethodManager) activity.getSystemService(Context.INPUT_METHOD_SERVICE);
         inputManager.hideSoftInputFromWindow(editText.getWindowToken(), HIDE_NOT_ALWAYS);
         Intent intent = new Intent(activity, OneScanActivity.class);
+        intent.putExtra(Constants.Config.ENABLE_BATCH_SCAN, Boolean.parseBoolean(widgetArgs.getJsonObject().optString(Constants.Config.ENABLE_BATCH_SCAN, "false")));
```

Comment:
Could we use `optBoolean(Constants.Config.ENABLE_BATCH_SCAN)` in place of `Boolean.parseBoolean(widgetArgs.getJsonObject().optString(Constants.Config.ENABLE_BATCH_SCAN, "false"))`?

The boolean value doesn't have to be a String in the widget i.e. convert "true" to true.

Reply:
fixed.